twake/frontend/src/app/components/FileUploads/FileUploadService.ts

```diff
@@ -58,7 +58,11 @@ export class FileUploadService extends EventEmitter {
 
       // Then we overwrite the file object with resumable
       pendingFile.resumable = this.getResumableInstance({
-        target: FileUploadAPIClient.getRoute({ companyId, fileId: pendingFile.backendFile.id }),
+        target: `${FileUploadAPIClient.getRoute({
```

Comment:
You do not need template string here. `getRoute` already returns a string

Reply:
You traumatised everyone with template strings :p

Reply:
Removed 😄